### PR TITLE
004 branding

### PR DESCRIPTION
**Pull request by ThomasS-kainos**

### CODE REVIEW

**Comment by kajatraczykowska** [comment]
Since all checks pass for now, should we let this get pushed up to main and then i'll work on fixing and updating the pa11y file since then hopefully i will be able to get it to point to the aws frontend stuff?

**Review by kajatraczykowska — APPROVED** [review]

File: views/homepage.html [hunk 1]
```diff
@@ -0,0 +1,123 @@
...
+
+    <title>Kainos Jobs Homepage</title>
+
+    <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.3.3/dist/css/bootstrap.min.css" rel="stylesheet"
+        integrity="sha384-QWTKZyjpPEjISv5WaRU9OFeRpok6YctnYmDr5pNlyT2bRjXh0JMhjY6hW+ALEwIH" crossorigin="anonymous">
+    <link href="./css/style.css" rel="stylesheet">
+</head>
+
+<body>
+    <nav class="navbar navbar-expand-lg shadow-sm">
```
Comment (shaunganley) [inline views/homepage.html]: move to a nav.html file and import so we can reuse on other pages

File: views/homepage.html [hunk 2]
```diff
@@ -0,0 +1,123 @@
...
+                        <a href="#" class="btn bg-primary text-white">About Us</a>
+                    </div>
+                </div>
+            </div>
+        </div>
+    </div>
+
+    <!--Main Content-->
+
+    <footer class="shadow-sm bg-primary text-light">
```
Comment (shaunganley) [inline views/homepage.html]: move to a footer.html file and import so we can reuse on other pages

**Review by shaunganley — CHANGES_REQUESTED** [review]

**Comment by ThomasS-kainos** [comment]
Moved the Nav and Footer to a template and the content/ "Body" of the page is now its own file in the pages folder. So now new pages can be put in a content block and the page can be called by the renderer and it all gets combined.

**Review by PhilipGardinerK — APPROVED** [review]

File: views/snippits/nav.html [hunk 3]
```diff
@@ -0,0 +1,10 @@
+<nav class="navbar navbar-expand-lg shadow-sm">
+    <img src="./images/Kainos 800x250 Logo (Transparent BG).png" width="200" height="63" alt="Kainos Compay Logo">
+    <div>
+        <div class="navbar-nav mt-4">
+            <a class="nav-item nav-link active" href="/">Home</span></a>
+            <a class="nav-item nav-link active" href="/">Jobs</span></a>
+            <a class="nav-item nav-link active" href="/">About Us</span></a>
```
Comment (shaunganley) [inline views/snippits/nav.html]: should these all be active?

**Review by shaunganley — APPROVED** [review]
snippets is misspelled

**Pull request merged**